### PR TITLE
Yarn CI command lists outdated packages

### PR DESCRIPTION
**Pull request by ignasbernotas**
screen shot 2018-09-27 at 16 07 38

### CODE REVIEW

File: util_scripts/outdated-dependencies.js [hunk 1]
```diff
@@ -29,6 +29,18 @@ const printPackageInfo = ({ name, current, latest }) => {
   console.log('\x1b[31m%s\x1b[0m - current: %s, latest: %s', name, current, latest)
 }
 
+const listOutdatedPackages = (packages) => {
+  packages.data.body.forEach((pkg) => {
+    if (needsToBeUpdated(pkg[1], pkg[3])) {
```
Comment (donce) [inline util_scripts/outdated-dependencies.js]: pkg[1], pkg[3], pkg[0] isn't quite readable, I would extract them to named variables.

File: util_scripts/outdated-dependencies.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  return false
+}
+
+const printPackageInfo = ({ name, current, latest }) => {
+  console.log('\x1b[31m%s\x1b[0m - current: %s, latest: %s', name, current, latest)
+}
+
+const listOutdatedPackages = (packages) => {
+  packages.data.body.forEach((pkg) => {
```
Comment (shroomist) [inline util_scripts/outdated-dependencies.js]: how about doing it more pure?
```js
const getOutdatedPackagesString = (packages) => {
return packages.data.body
    .filter((pkg) => needsToBeUpdated(pkg[1], pkg[3]))
    .map((pkg) => beautifySerialize{
         name: pkg[0],
         current: pkg[1],
         latest: pkg[3]
       })
    .join('\n')

}
...
const beautifySerialize = ({ name, current, latest }) => {
   return `\x1b[31m%${name}x1b[0m - current: ${current}, latest: ${latest}`
 }
const packages = JSON.parse(json)
const outdated = getOutdatedPackages(packages)
console.log(outdated)

Comment (ignasbernotas) [reply util_scripts/outdated-dependencies.js]: This is an equivalent to a bash script, don't see much point in doing this

Comment (shroomist) [reply util_scripts/outdated-dependencies.js]: ... but I've tried so hard coding it in github.. 😿

**Comment by ignasbernotas** [comment]
@shroomist @donce please review again:)

**Pull request merged**